pkg/status/render/render.go

```diff
@@ -31,28 +26,6 @@ func FormatStatus(data []byte) (string, error) {
 	if renderError != "" || err != nil {
 		return renderError, err
 	}
-	forwarderStats := stats["forwarderStats"]
-	if forwarderStatsMap, ok := forwarderStats.(map[string]interface{}); ok {
-		forwarderStatsMap["config"] = stats["config"]
-	} else {
-		log.Warn("The Forwarder status format is invalid. Some parts of the `Forwarder` section may be missing.")
-	}
```

Comment:
I aggregated all forwarded stats within the same function when fetching the status.
https://github.com/DataDog/datadog-agent/blob/30cff43511084476ccc50299f3171ce6b0414b67/pkg/status/status.go#L320-L327